ts/component/table-builder.ts

```diff
@@ -15,7 +15,7 @@ import { showErrorDialog } from "./dialog";
 import { updateUrlParams } from "../util/urls";
 
 type SortType = "number" | "text" | "date";
-type FilterType = "text" | "dropdown";
+type FilterType = "text" | "dropdown" | "date";
```

Comment:
"text" can be used for the dates instead of adding this option. As is, it would cause an error if there were a date filter and the filters were not hidden

Reply:
From what I understand, it would throw an error if filterType is "text" and has no autocomplete url. Should I remove the check? I thought it would be good to keep it for other filters

Reply:
Fair enough. Will do